pkg/graphviz/traceflow.go

```diff
@@ -422,3 +488,37 @@ func GenGraph(tf *crdv1alpha1.Traceflow) (string, error) {
 
 	return genOutput(graph, false), nil
 }
+
+func getCapturedPacketLabel(tf *crdv1alpha1.Traceflow) string {
+	label := "caputredPacket:\\ldstIP:" + tf.Status.CapturedPacket.DstIP + "\\l"
+
+	if tf.Status.CapturedPacket.IPHeader != (crdv1alpha1.IPHeader{}) {
+		label = label + "ipHeader: " + "\\l" +
+			"    flags: " + strconv.Itoa(int(tf.Status.CapturedPacket.IPHeader.Flags)) + "\\l" +
```

Comment:
IMO you should use `fmt.Sprintf` instead of these `strconv.Itoa(int(...))` calls